### PR TITLE
Adding scroll-timeline css at rule

### PR DESCRIPTION
**Pull request by Rumyra**
#### Summary

This is available in Firefox 97 - see https://bugzilla.mozilla.org/show_bug.cgi?id=1676791 or/and https://bugzilla.mozilla.org/show_bug.cgi?id=1676782 

This is available in Chrome, but I'm unsure of the version it was first available in. If anyone points me in the right direction I'm happy to add

#### Test results and supporting details

Passes tests

#### Related issues

Part of https://github.com/mdn/content/issues/9368

### CODE REVIEW

**Comment by ddbeck** [comment]
It looks like Chrome is behind a flag and not yet finished: https://bugs.chromium.org/p/chromium/issues/detail?id=1074052

File: css/at-rules/scroll-timeline.json [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "orientation": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@scroll-timeline",
```
Comment (ddbeck) [inline css/at-rules/scroll-timeline.json:55]: If the URL is not different from the parent feature, I'd omit it.

```suggestion
```

File: css/at-rules/scroll-timeline.json [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "scroll-offsets": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@scroll-timeline",
```
Comment (ddbeck) [inline css/at-rules/scroll-timeline.json:104]:
```suggestion
```

File: css/at-rules/scroll-timeline.json [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "source": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@scroll-timeline",
```
Comment (ddbeck) [inline css/at-rules/scroll-timeline.json:153]:
```suggestion
```

File: css/at-rules/scroll-timeline.json [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```
Comment (ddbeck) [inline css/at-rules/scroll-timeline.json:48]: These are all single-implementation right now, so let's set `experimental` to true. See the [_Choosing an experimental status_ data guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status) for details.

```suggestion
            "experimental": true,
```

File: css/at-rules/scroll-timeline.json [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (ddbeck) [inline css/at-rules/scroll-timeline.json:194]:
```suggestion
              "experimental": true,
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
A few suggestions for you, Ruth.

**Comment by Rumyra** [comment]
Thanks loads @ddbeck 👍

**Pull request merged**